common/db.go

```diff
@@ -379,9 +390,29 @@ func (db *DB) InsertUnbondingLock(id *big.Int, delegator ethcommon.Address, amou
 	return nil
 }
 
+// DeleteUnbondingLock deletes an unbonding lock from the DB with the given ID and delegator address. This method will
```

Comment:
Perhaps it's better to start the new sentence on a new line
```
// DeleteUnbondingLock deletes an unbonding lock from the DB with the given ID and delegator address. 
// This method will return nil for non-existent unbonding lock
```

Reply:
Fixed in 7ef1c97